### DISCUSSION TITLE
⏳ 🔗 Conditional Fields for Elementor Forms

### DISCUSSION THREAD

**Discussion by israrmansuri15** [elementor/elementor#21722] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Elementor form is great but if we have a conditional field option in Elementor that will be a huge plus.

### Describe the solution you'd like

Elementor Form should have an option for conditional field

**Comment by awebguy99** [comment]
That would be great. This has been asked for ages, hopefully it's on their list of features to add.

**Comment by ausdemnichts** [comment]
I hope that this feature finally comes. It's a fundamental functionality for many websites.

**Comment by fabigrafo** [comment]
+1, So much needed. But the longer it takes, the less hope I have.

**Comment by ayoubkhan558** [comment]
+1

**Comment by maurocarreraok** [comment]
+1

**Comment by fabigrafo** [comment]
It’s Elementor, be realistic. 😬

**Comment by wp-add-ons** [comment]
I found a free plugin that can do this. Why doesn't Elementor update?
Download: https://wordpress.org/plugins/conditional-logic-for-elementor-forms/

**Comment by narinder9** [comment]
Hi Everyone 👋,

 I was also **waiting for Conditional Fields support** in Elementor Forms. Since this feature is still not available, we decided to **build a solution** ourselves!

We have created a **free plugin** : [Conditional Fields for Elementor Form](https://wordpress.org/plugins/conditional-fields-for-elementor-form/)

With this plugin, you can **show or hide form field**s based on values entered in other fields using simple "if condition" logic.

> For example: If option A is selected, show Field XYZ. If not, keep it hidden.

👉 [Check out the live demo](https://coolplugins.net/product/conditional-fields-for-elementor-form/?utm_source=elementor_repo&utm_medium=issue_comment&utm_campaign=demo&utm_content=21722#demos) 


Give it a try and let us know your feedback — I’ll keep improving it based on your suggestions.

Thanks in Advance!
